apps/arweave/src/ar_block_cache.erl

```diff
@@ -690,13 +688,13 @@ test_block_cache() ->
 	add(bcache_test, B2),
 	mark_tip(bcache_test, block_id(B2)),
 	add(bcache_test, B2_2 = on_top(random_block(1), B2)),
-	?assertMatch({B1_2, [B1], {{not_validated, ExpectedStatus}, _Timestamp}},
+	?assertMatch({B1_2, [B1], {{not_validated, ExpectedStatus}, _}},
```

Comment:
ditto above. I don't think we should be removing context information from Erlang code and should leave it up to the engineer(s) maintaining that section of code what is most intelligible.

Reply:
I don't fully understand why I removed this there, maybe I was on auto-pilot, but elvis has no problem with unused underscore variables in most places

example on line 124

![2023-12-08 15 28 31](https://github.com/ArweaveTeam/arweave/assets/6074754/21e26898-82b5-4288-a213-e0fd4e847f5b)

Reply:
reverted